CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Contributing to License List Data
+
+Technical questions or issues can be sent to the [SPDX technical team mailing list](mailto:spdx-tech@lists.spdx.org).
+Questions or issues on the license content can be sent to the [SPDX legal team mailing list](mailto:spdx-legal@lists.spdx.org).
+
+Note that the content of this repository is generated by the [SPDX tools](http://github.com/spdx/tools) from the [license list source](https://github.com/spdx/license-list-XML).
+
+As far as the *content* of the license information is concerned, see the [license-list-XML Contributing](https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md) document for information on how to report issues, request new licenses, or contribute changes.
```

Comment:
I think you should move “the” and “document” inside the link, so:

```markdown
…see [the license-list-XML contributing document](https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md) for …
```

And I'm ambivalent on “document” vs. “documentation”, but whichever you use, you should probably also use in the README if you use similar phrasing there.